packages/xrpl/src/models/transactions/MPTokenIssuanceCreate.ts

```diff
@@ -0,0 +1,143 @@
+import { ValidationError } from '../../errors'
+import { isHex } from '../utils'
+
+import { BaseTransaction, GlobalFlags, validateBaseTransaction } from './common'
+import type { TransactionMetadataBase } from './metadata'
+
+const SANITY_CHECK = /^[0-9]+$/u
```

Comment:
nit: can we rename this to indicate that its checking for a number format, and move it to the utils?